ui/lib/kv/addon/components/page/secret/overview.hbs

```diff
@@ -101,6 +101,6 @@
   <KvPathsCard @backend={{@backend}} @path={{@path}} @isCondensed={{true}} />
 </Hds::Card::Container>
 
-{{#if @subkeys.subkeys}}
+{{#if (and @subkeys.subkeys this.version.isEnterprise)}}
```

Comment:
We don't need this check anymore then right?

Reply:
Yes!